opencontrail/config.sls

```diff
@@ -154,7 +154,7 @@ publisher_init:
     - service: opencontrail_config_services
 {%- endif %}
 
-{%- if not grains.get('virtual_subtype', None) == "Docker" %}
+{%- if not common.vendor == "juniper" or not grains.get('virtual_subtype', None) == "Docker" %}
```

Comment:
this is not obsolate, as we really don't want to remove that file for juniper packaging..